### ISSUE TITLE
StaveTempo not being rendered correctly

### ISSUE THREAD

**Issue by brunostuani** [0xfe/vexflow#1586]
Hi! Is there any particular reason that this might be happening?

<img width="810" alt="tempo-issue" src="https://github.com/0xfe/vexflow/assets/27310471/21261410-78ef-499a-a686-567eb2d82c78">

I'm adding a stave tempo using this snippet:

```
    if (!!layout.bpm) {
      this.stave.setTempo({
        bpm: layout.bpm,
        duration: '4',
      }, 0);
    }
```

And the stave is being initialized with:

```
    this.stave = new Stave(x, y, width,
      {
        space_above_staff_ln: 0,
        space_below_staff_ln: 0,
      }
    );
```

Thank you

**Comment by rvilarl** [comment]
Could you provide a running jsfiddle?

**Comment by brunostuani** [comment]
Apologies, I should have done that sooner. Here it is:
https://jsfiddle.net/xc9o1rjf/5/

**Comment by rvilarl** [comment]
The issue is already resolved in 4.2.0 see  https://jsfiddle.net/txeLzp1h/1/

The last release is 4.2.2 , I would recommend to use this one.

**Comment by brunostuani** [comment]
Thanks! It did fix my problem. However it caused another one, I'll open a different bug for it. Thanks
